### PR TITLE
Add metrics API and redefine fetching metrics selection to redux

### PR DESCRIPTION
**Pull request by Sami1309**

### CODE REVIEW

File: dashboard/src/api/resources/Metrics.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  fetchMetrics(instance) {
+    var fetchAddress = `${METRICS_API_URL}/metadata`;
+    return fetch(fetchAddress, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => res.json().then((json_data) => json_data))
```
Comment (simba-git) [inline dashboard/src/api/resources/Metrics.js:25]: ".then((json_data) => json_data)" this seems like an identity lambda that doesn't do anything, am I missing something?

File: dashboard/src/api/resources/Metrics.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default class MetricsAPI {
+  fetchInstances() {
+    var fetchAddress = `${METRICS_API_URL}/label/__name__/values`;
+
+    return fetch(fetchAddress, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((res) => res.json().then((json_data) => json_data))
```
Comment (simba-git) [inline dashboard/src/api/resources/Metrics.js:12]: Same as below *

**Review by simba-git — CHANGES_REQUESTED** [review]
One question

File: dashboard/src/api/resources/Metrics.js (hunk 1 above)
Comment (Sami1309) [reply dashboard/src/api/resources/Metrics.js:25]: yea it's an artifact left over from debugging

**Pull request merged**